pkg/sql/sem/tree/backup.go

```diff
@@ -28,10 +28,15 @@ const (
 	// backup is not said to have complete table coverage unless it was created
```

Comment:
I wonder whether we should change the comment above `type DescriptorCoverage` now that we're using this enum a little more liberally. Something like:

// DescriptorCoverage specifies the subset of descriptors that are requested during a backup
// or a restore.

Reply:
Done.